src/uadk.h

```diff
@@ -40,10 +40,22 @@ void uadk_e_destroy_dh(void);
 int uadk_e_bind_ecc(ENGINE *e);
 void uadk_e_destroy_ecc(void);
 int uadk_e_is_env_enabled(const char *alg_name);
+void uadk_e_set_env_enabled(const char *alg_name, unsigned int value);
```

Comment:
Doed the declaration of uadk_e_set_env_enabled() needs to be placed in uadk_utils.h file ?